packages/ui-components/package.json

```diff
@@ -39,6 +39,7 @@
   ],
   "devDependencies": {
     "@agoric/eslint-config": "^0.4.0",
+    "@agoric/swingset-liveslots": "^0.10.2",
```

Comment:
this doesn't really depend on that package right? you're trying to fix some version incompatibility. I think `resolutions` is the tool for that.
https://github.com/Agoric/ui-kit/blob/cd7ee7620638c6970651d557ee7962fe083ed25a/package.json#L46-L49